<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userMode</class>
 <widget class="QDialog" name="userMode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="genreEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="suggestionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Suggestion</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="watchedEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Watched Movie</string>
   </property>
  </widget>
  <widget class="QComboBox" name="liked">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="movieLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="trailerButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Watch Trailer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Movie</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Movie</string>
   </property>
  </widget>
  <widget class="QListWidget" name="items">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>411</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open in App</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="undoButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>61</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
